packages/clay-management-toolbar/src/DefaultState.tsx

```diff
@@ -0,0 +1,258 @@
+							label={
+								!filterItems && !sortingOrder && !showSearch
+									? labels.checkbox
+									: undefined
+							}
+							onChange={onCheckboxChange}
+						/>
+					</Item>
+				)}
+
+				{filterItems && (
+					<ClayDropDownWithBasicItems
+						items={filterItems}
+						spritemap={spritemap}
+						trigger={
+							<ClayButton
+								className="nav-link"
+								displayType="unstyled"
+							>
+								<span className="navbar-breakpoint-down-d-none">
+									<span className="navbar-text-truncate">
+										{'Filter and Order'}
+									</span>
+
+									<ClayIcon
+										className="inline-item inline-item-after"
+										spritemap={spritemap}
+										symbol="caret-bottom"
+									/>
+								</span>
+								<span className="navbar-breakpoint-d-none">
+									<ClayIcon
+										spritemap={spritemap}
+										symbol="filter"
+									/>
+								</span>
+							</ClayButton>
+						}
+					/>
+				)}
+
+				{sortingOrder && (
+					<Item>
+						{sortingURL && !disabled ? (
+							<ClayLink
+								className={classNames(
+									'nav-link nav-link-monospaced',
+									{
+										'order-arrow-down-active':
+											sortingOrder === 'desc',
+										'order-arrow-up-active':
+											sortingOrder === 'asc',
+									}
+								)}
+								href={sortingURL}
+								title={titlesLang.sorting}
+							>
+								<ClayIcon
+									spritemap={spritemap}
+									symbol="order-arrow"
+								/>
+							</ClayLink>
+						) : (
+							<ClayButton
+								className={classNames(
+									'nav-link nav-link-monospaced',
+									{
+										'order-arrow-down-active':
+											sortingOrder === 'desc',
+										'order-arrow-up-active':
+											sortingOrder === 'asc',
+									}
+								)}
+								displayType="unstyled"
+								onClick={onSortingButtonClick}
+								title={titlesLang.sorting}
+							>
+								<ClayIcon
+									spritemap={spritemap}
+									symbol="order-arrow"
+								/>
+							</ClayButton>
+						)}
+					</Item>
+				)}
+			</ItemList>
+
+			{showSearch && (
+				<div
+					className={classNames('navbar-form navbar-form-autofit', {
+						'navbar-overlay navbar-overlay-sm-down': !onlySearch,
+						show: searchMobile,
+					})}
+				>
+					<SearchForm onlySearch={onlySearch}>
+						<SearchInput
+							disabled={disabled}
+							onCloseButtonClick={() => setSearchMobile(false)}
+							onValueChange={onSearchValueChange}
+							onlySearch={onlySearch}
+							spritemap={spritemap}
+							value={searchValue}
+						/>
+					</SearchForm>
+				</div>
+			)}
+
+			<ItemList>
+				{showSearch && !onlySearch && (
+					<Item className="navbar-breakpoint-d-none">
+						<ClayButton
+							className="nav-link nav-link-monospaced"
+							displayType="unstyled"
+							onClick={() => setSearchMobile(true)}
+						>
+							<ClayIcon spritemap={spritemap} symbol="search" />
+						</ClayButton>
+					</Item>
+				)}
+
+				{showInfoButton && (
```

Comment:
This will always evaluate to true since we have a default no-op.

Reply:
You say about that if we keep checking `onInfoButtonClick` right?